README.md

```diff
@@ -118,6 +118,7 @@ You can overwrite Rollbar configuration in environment's config. Here is the def
         source_map_enabled: true,
         guess_uncaught_frames: true
         // code_version: "some version string, such as a version number or git sha",
```

Comment:
`code_version: YOUR_APP_VERSION` // by default we are using app version + git hash (v2.3.4-123FDDF)

Reply:
What do you mean? The current version, by default, doesn't provide value for this field.